File: trunk/download/stream/src/test/java/org/simpleframework/xml/transform/math/BigIntegerTransformTest.java
package org.simpleframework.xml.transform.math;

import java.math.BigInteger;
import junit.framework.TestCase;

public class BigIntegerTransformTest extends TestCase {
   
   public void testBigInteger() throws Exception {
      BigInteger integer = new BigInteger("1");
      BigIntegerTransform format = new BigIntegerTransform();
      String value = format.write(integer);
      BigInteger copy = format.read(value);
      
      assertEquals(integer, copy);      
   }
}

File: trunk/download/stream/port/src/main/Xml/Transform/BigIntegerTransformTest.cs
#region Using directives
using SimpleFramework.Xml.Transform;
using SimpleFramework.Xml;
using System;
#endregion
package SimpleFramework.Xml.transform;
public class BigIntegerTransformTest : TestCase {
   public void TestBigInteger() {
      BigInteger integer = new BigInteger("1");
      BigIntegerTransform format = new BigIntegerTransform();
      String value = format.write(integer);
      BigInteger copy = format.read(value);
      AssertEquals(integer, copy);
   }
}
}
